apis/twitter/twitter_api/twitter_api.ecf

```diff
@@ -11,16 +11,16 @@
 		<option warning="true">
 			<assertions precondition="true" postcondition="true" check="true" invariant="true" loop="true" supplier_precondition="true"/>
 		</option>
+		<setting name="console_application" value="true"/>
 		<capability>
 			<concurrency support="none"/>
```

Comment:
Is that true that only "none" concurrency is supported?
That is an issue, as it could not be used in concurrent application.

I suspect that settings is wrong, and that the lib is scoop capable.

Reply:
Yes that's wrong.